java/ql/src/Security/CWE/CWE-347/MissingJWTSignatureCheck.qhelp

```diff
@@ -25,9 +25,9 @@ by overriding the <code>onPlaintextJws</code> or <code>onClaimsJws</code> of <co
 <example>
 
```

Comment:
Typos in line 7: s/Jws/Jwt/

Reply:
also in line 20-21

Reply:
`io.jsonwebtoken.jjwt` also looks suspiciously like a typo.

Reply:
Why do you think this is a typo?
Maybe there is something else that could be improved in the qhelp file, because these are no typos.

I think it would have been better if I had clarified the difference between JWT (unsigned) and JWS (signed) when I created the file.
Unfortunately, JWT and JWS are very often used synonymously.

Reply:
For clarification, because sometimes it can get confusing with so many acronyms:

* JWT stands for JSON Web Token: https://datatracker.ietf.org/doc/html/rfc7519
* JWS stands for JSON Web Signature: https://datatracker.ietf.org/doc/html/rfc7515
* `jjwt` is a Java library for JWT : https://github.com/jwtk/jjwt

Reply:
I'm just pattern matching - I don't actually know anything about JWT, so I just assumed that Jws and jjwt were typos for jwt.